site/src/pages/TemplatePage/TemplateEmbedPage/TemplateEmbedPage.tsx

```diff
@@ -55,11 +59,16 @@ function getClipboardCopyContent(
 	const deploymentUrl = `${window.location.protocol}//${window.location.host}`;
 	const createWorkspaceUrl = `${deploymentUrl}/templates/${organization}/${templateName}/workspace`;
 	const createWorkspaceParams = new URLSearchParams(buttonValues);
+	if (createWorkspaceParams.get("name") === "") {
+		createWorkspaceParams.delete("name"); // skip default workspace name if undefined
```

Comment:
```suggestion
		createWorkspaceParams.delete("name"); // skip default workspace name if empty
```

we are turning it into `undefined` here by deleting it. maybe that's what you meant? but undefined isn't the condition here, it's the desired end value